install-haproxy.sh

```diff
@@ -5,15 +5,15 @@ apk update
 
 HAPROXY_VERSION=${HAPROXY_VERSION:-1.6.6}
 HAPROXY_MAJOR_VERSION=${HAPROXY_VERSION:0:3}
-LIBSLZ_VERSION=v1.0.0
+LIBSLZ_VERSION=v1.1.0
 BUILD_DEPS="curl make gcc g++ linux-headers python pcre-dev openssl-dev lua5.3-dev"
 RUN_DEPS="pcre libssl1.0 musl libcrypto1.0 busybox lua5.3-libs"
 
 # install build dependencies
 apk add --virtual build-dependencies ${BUILD_DEPS}
 
 # Compile libslz
-curl -OJ "http://git.1wt.eu/web?p=libslz.git;a=snapshot;h=v1.0.0;sf=tgz"
+curl -OJ "http://git.1wt.eu/web?p=libslz.git;a=snapshot;h=${LIBSLZ_VERSION};sf=tgz"
```

Comment:
i looked around -- it looks like the author chose a hosting name and certs that do not match.

Reply:
we should, at least, sha256sum it offline and check it. (it's imperfect -- a small step better).